### ISSUE TITLE
Question about SAC benchmarking on MuJoCo

### ISSUE THREAD

**Issue by hnyu** [aai-institute/tianshou#487]
Hi, really nice repo. I was looking for benchmark results of SAC on MuJoCo locomotion tasks and found this table:

https://github.com/thu-ml/tianshou/tree/master/examples/mujoco#sac

As I can see in the provided example file 

https://github.com/thu-ml/tianshou/blob/master/examples/mujoco/mujoco_sac.py

the default learning rates for both the actor and critic are set to 1e-3. And I can also see a sentence in the readme file saying that Tianshou's SAC results indeed used 1e-3: "TD3's learning rate is set to 3e-4 while it is 1e-3 for DDPG/SAC." 

However, the original SAC paper says in their appendix that they used 3e-4 for all parameters instead. I think in order to compare with the paper's results, 3e-4 will be a more fair hyperparameter. Or a comparison between 3e-4 and 1e-3 is needed to see how largely the results are affected.

**Comment by ChenDRAG** [comment]
Thanks a lot for your information. From what I remember, we used lr 1e-3 in order to be consistent with OPENAI spinningup , and this parameter is never tuned. Orginal paper of SAC does use lr 3e-4 as its default. Both two learning rates achieve good (or similar) results in most Mujoco environments, so I personally assume that SAC is not that sensitive to lr in this case. However, if you are interested, feel free to run
`python mujoco_sac.py --learning_rate=3e-4`
to see if it makes any difference.

**Comment by hnyu** [comment]
> Thanks a lot for your information. From what I remember, we used lr 1e-3 in order to be consistent with OPENAI spinningup , and this parameter is never tuned. Orginal paper of SAC does use lr 3e-4 as its default. Both two learning rates achieve good (or similar) results in most Mujoco environments, so I personally assume that SAC is not that sensitive to lr in this case. However, if you are interested, feel free to run `python mujoco_sac.py --learning_rate=3e-4` to see if it makes any difference.

OK I see. Thanks for your quick reply.
